### ISSUE TITLE
Terminal application cannot find spellfix.so

### ISSUE THREAD

**Issue by aulin** [laurent22/joplin#3783]
After updating the joplin terminal application to version 1.1.7, when I try to run it, I get the following error:

```
❯ joplin
events.js:291
      throw er; // Unhandled 'error' event
      ^

[Error: SQLITE_ERROR: ./build/lib/sql-extensions/spellfix.so: cannot open shared object file: No such file or directory
Emitted 'error' event on Database instance at:
] {
  errno: 1,
  code: 'SQLITE_ERROR'
}
```

The file actually resides in `~/.joplin-bin/lib/node_modules/joplin/lib/sql-extensions/spellfix.so` but that's not the path that gets reported. If I create the folder structure in the current directory with a symlink to the so-file, joplin starts just fine.

## Environment

Joplin version: 1.1.7
Platform: terminal
OS specifics: Ubuntu 15.04.5 LTS x86_64
Node version: 14.11.0
Npm version: 6.14.8

## Steps to reproduce

1. Update the joplin terminal app by doing `NPM_CONFIG_PREFIX=~/.joplin-bin npm install -g joplin`
2. Run the joplin terminal app

## Describe what you expected to happen

Joplin should launch normally.

## Logfile

File .config/joplin/log.txt:
`2020-09-21 14:29:35: "Profile directory: /home/johan/.config/joplin"`

**Comment by duckunix** [comment]
I am seeing this as well on Ubuntu 20.04 x86_64 and npm 6.14.4

**Comment by aulin** [comment]
I updated my post with node/npm versions as well.

**Comment by aulin** [comment]
On my Raspberry Pi, the app doesn't start even with a directory structure containing spellfix.so in the current dir. That instance is installed in my home folder using nvm though, so it's installed using:

`npm i -g joplin`

and ends up in:

`/home/johan/.nvm/versions/node/v14.8.0/bin/joplin`

The environment for that computer is:

Joplin version: 1.1.7
Platform: terminal
OS specifics: Manjaro ARM Linux aarch64
Node version: 14.8.0
Npm version: 6.14.7

**Comment by seth100** [comment]
Same here. `Pop!_OS 20.04 LTS x64` and `npm 6.14.8`

**Comment by yashasolutions** [comment]
same here 
Linux pop-os 5.0.0-21-generic #22+system76-Ubuntu 
npm 6.13.0

**Comment by dbarrosop** [comment]
Same on macos

**Comment by rusty-electron** [comment]
facing the same issue on arch kernel 5.8.10

**Comment by yashasolutions** [comment]
Fixed with : 
`$ sudo apt-get install libsqlite3-dev`

then reinstalled joplin, and worked.

**Comment by duckunix** [comment]
Actually, it was commit 6aca233 which fixed it.  I already had the latest version of libsqlite3-dev installed.

**Comment by aulin** [comment]
I can confirm that the issue is resolved in version 1.2.1 for both my Ubuntu laptop and my Manjaro raspberry pi.
